### PR TITLE
LPS-144635 Prune the CSS after converting to ClayTreeview

### PR DESCRIPTION
**Pull request by pat270**
https://issues.liferay.com/browse/LPS-144635

This pr covers the CSS in:

modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss
modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss

from the list in the issue.

product-menu

@julien

### CODE REVIEW

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:relevant

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss [hunk 1]
```diff
@@ -3,92 +3,53 @@
...
-				border-left: none;
-				padding-left: 22px;
-
-				.lfr-treeview-node-list {
-					border-left: 1px solid $cadmin-gray-800;
-					margin-bottom: 6px;
-					margin-top: 6px;
+			.treeview-dark {
+				> .treeview-item > .treeview-link {
+					color: $cadmin-white;
```
Comment (sandrodw3) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss:11]: I think this line is not needed, as only selected nodes are white in the current design 🙂

**Review by sandrodw3 — COMMENTED** [review]
Hi @pat270! This looks good to me 🥰
I just left a comment regarding a line that I think we should remove. As you marked the PR as draft, I don't know if you are planning to add something else. Apart of the comment I left, in my opinion it's good to go 🙂

**Comment by pat270** [comment]
@sandrodw3 Thanks for looking at this. I'm going to close this since there is a feature flag that enables the old Treeview. We need to find a way to provide one or the other.

**Pull request closed without merging**